test-e2e/local-peers.js

```diff
@@ -11,6 +11,9 @@ test('Local peers discovery each other and share device info', async (t) => {
   const desktopManagers = await createManagers(5, t, 'desktop')
   const managers = [...mobileManagers, ...desktopManagers]
   connectPeers(managers, { discovery: true })
+  t.teardown(() => {
+    disconnectPeers(managers)
+  })
```

Comment:
Nit: I think this callback should return a promise. One of these should work:

```suggestion
  t.teardown(async () => {
    await disconnectPeers(managers)
  })
```

...or:

```suggestion
  t.teardown(() => disconnectPeers(managers))
```

The latter doesn't _immediately_ work because the callback to `t.teardown()` is expected to return `void | Promise<void>`, but `disconnectPeers` returns `Promise<void[]>`. I'd fix that by updating `test-e2e/utils.js` like this:

```diff
diff --git a/test-e2e/utils.js b/test-e2e/utils.js
index e5fd403..7dd5575 100644
--- a/test-e2e/utils.js
+++ b/test-e2e/utils.js
@@ -24,7 +24,7 @@ const clientMigrationsFolder = new URL('../drizzle/client', import.meta.url)
  * @param {readonly MapeoManager[]} managers
  */
 export async function disconnectPeers(managers) {
-  return Promise.all(
+  await Promise.all(
     managers.map(async (manager) => {
       return manager.stopLocalPeerDiscovery({ force: true })
     })
```

Reply:
Not sure if it works for us, but we might be able to [enable a "no floating promises" ESLint rule](https://typescript-eslint.io/rules/no-floating-promises/) to help with this kind of problem.